src/openrct2-ui/windows/Ride.cpp

```diff
@@ -5992,25 +6000,67 @@ static void window_ride_graphs_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi
             case GRAPH_VERTICAL:
                 top = measurement->vertical[x] + 39;
                 bottom = measurement->vertical[x + 1] + 39;
+                redThresholdPositive = (RIDE_G_FORCES_RED_POS_VERTICAL / 8) + 39;
+                redThresholdNegative = (RIDE_G_FORCES_RED_NEG_VERTICAL / 8) + 39;
                 break;
             case GRAPH_LATERAL:
                 top = measurement->lateral[x] + 52;
                 bottom = measurement->lateral[x + 1] + 52;
+                redThresholdPositive = (RIDE_G_FORCES_RED_LATERAL / 8) + 52;
+                redThresholdNegative = -(RIDE_G_FORCES_RED_LATERAL / 8) + 52;
                 break;
             default:
                 log_error("Wrong graph type %d", listType);
                 top = bottom = 0;
         }
 
+        // Adjust line to match graph widget position.
         top = widget->bottom - widget->top - top - 13;
         bottom = widget->bottom - widget->top - bottom - 13;
+
+        // If red threshold is supported, adjust as well. Otherwise, keep 0.
+        if (!(redThresholdPositive == redThresholdNegative == 0))
```

Comment:
Afaik, this gets parsed as `!((redThresholdPositive == redThresholdNegative) == false)` aka `!(!(redThresholdPositive == redThresholdNegative))`` aka `redThresholdPositive == redThresholdNegative`. Is this what you meant, or did you mean `redThresholdPositive != 0 && redThresholdNegative != 0`?

Reply:
Fixed

Reply:
You changed a different if statement. The one @SijmenSchoon pointed out still remains as it was.